cms/templates/js/self-paced-due-date-editor.underscore

```diff
@@ -1,24 +1,30 @@
-<ul class="list-fields list-input date-setter">
-    <li class="field field-text field-due-in">
-        <label for="due_in"><%- gettext('Due in:') %></label>
-        <input type="number" id="due_in" name="due_in" value=""
-            placeholder="" autocomplete="off" min="1" max="18"/> weeks
-    </li>
-</ul>
+<div id="relative_date_input">
+    <ul class="list-fields list-input date-setter">
+        <li class="field field-text field-due-in">
+            <!-- Translators: Please use a generic pluralization that makes sense in most contexts for the second half of the sentence "weeks from learner enrollment date" which is preceded by "Due in: [input entry]" -->
+            <label for="due_in"><%- gettext('Due in:') %></label>
+            <input type="number" id="due_in" name="due_in" value=""
+                placeholder="" autocomplete="off" min="1" max="18" style="width:20%"/>
+            <%- gettext('weeks from learner enrollment date')%>
```

Comment:
Is it possible to ngettext this or no? It might be too complicated with the value you want to know is dynamic

Reply:
I attempted to but wasn't able to:( I think it is a bit too complicated with trying to listen to the input in the same underscore file

Reply:
All good then